engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -472,11 +472,6 @@ static void LogFrameBufferError(GLenum status)
         return GL_FALSE;
     }
 
-    static bool OpenGLIsSupported()
-    {
-        return Initialize();
-    }
-
```

Comment:
This was changed to OpenGLIsInitialize since otherwise the "Initialize" function will select an adapter by priority instead of a specific device.